mozci/task.py

```diff
@@ -82,6 +100,29 @@ class TestTask(Task):
     _errors: List = field(default=None)
     _groups: List = field(default=None)
 
+    # XXX: Once bug 1613937 and bug 1613939 are fixed, we can remove the filtering
+    # and slash replacing, and turn the warning on bad group names into an assertion.
+    def __post_init__(self):
+        if self._groups is not None:
+            self._groups = [
+                group.replace("\\", "/")
+                for group in self._groups
+                if not is_bad_group(self.id, group)
+            ]
+
+        def update_group(result):
+            if result.group is not None:
+                result.group = result.group.replace("\\", "/")
```

Comment:
Might be worth inlining [this file](https://searchfox.org/mozilla-central/source/python/mozbuild/mozpack/path.py#77) to `mozci.util` and using the `normpath` function.

Reply:
It looks like it doesn't change `\\` to `/`: `posixpath.normpath(normsep("part1\\part2")) == "part1\\part2"`

Reply:
Oh right.. that function only does modifications if `os.sep != '/'`, and since this is running on Linux it's leaving it alone. We could modify it I guess but I'm fine with just leaving the replaces in if you want.

Reply:
Hopefully we can remove this post-processing soon anyway, once the bug is fixed (and after a few weeks to have enough new data).